frontend/src/utils/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Shared utilities file.
+ */
+
+const buildUrlEndingFromFilters = (filters) => {
+  let url = "&";
+  Object.keys(filters).map((filterKey) => {
+    if (filters[filterKey] && filters[filterKey].length > 0) {
+      if (Array.isArray(filters[filterKey]))
+        url += encodeURI(filterKey + "=" + filters[filterKey].join()) + "&";
+      else url += encodeURI(filterKey + "=" + filters[filterKey] + "&");
+    }
+  });
+  return url;
+};
+
+export { buildUrlEndingFromFilters };
```

Comment:
We have used the folder `/public/lib` for that so far. I think this function would fit there better.

Reply:
Nice, TIL. I'll move there 👍 Why is it named /public`?

Reply:
public is usually used for images and other static data because I wanted to keep src only for components. But thinking about it putting it in src actually makes much more sense. Maybe we can move it in the future (in a seperate PR)